tests/test_other.py

```diff
@@ -2187,6 +2187,12 @@ def test_file_packager_mention_FORCE_FILESYSTEM(self):
     err = self.run_process([EMCC, path_from_root('tests', 'hello_world.c'), '--preload-file', 'data.txt'], stdout=PIPE, stderr=PIPE).stderr
     self.assertEqual(len(err), 0)
 
+  def test_file_packager_returns_error_if_target_equal_to_jsoutput(self):
+    MESSAGE = 'error: TARGET should not be the same value of --js-output'
+    result = self.run_process([PYTHON, FILE_PACKAGER, 'test.data', '--js-output', 'test.data'], check=False, stdout=PIPE, stderr=PIPE)
```

Comment:
This should fix one of the CI errors:
```suggestion
    result = self.run_process([PYTHON, FILE_PACKAGER, 'test.data', '--js-output=test.data'], check=False, stdout=PIPE, stderr=PIPE)
```
That error message could be better too :rofl: Sorry about that.

Reply:
Ah, the irony :laughing: yeah i thought it was the first test.data and completely overlooked the rest :sweat_smile: Thanks a lot for the hint!